docs/modules/multivcfanalyzer.md

```diff
@@ -0,0 +1,8 @@
+---
+Name: MultiVCFAnalyzer
+URL: http://www.github.com/alexherbig/MultiVCFAnalyzer
+Description: >
+    MultiVCFAnalyzer collects multiple VCF files and outputs combined genotype calls in a number of file formats.
+---
+
+MultiVCFAnalyzer reads multiple VCF files as produced by the GATK UnifiedGenotyper and after filtering provides the combined genotype calls in a number of formats that are suitable for follow-up analyses such as phylogenetic reconstruction, SNP effect analyses, population genetic analyses etc.
```

Comment:
"MultiVCFanalyzer reads multiple VCF files as produced by GATK UnifiedGenotyper, performs filtering based on a number of criteria, and provides the combined genotype calls in a number of formats that are suitable for follow-up analyses such as phylogenetic reconstruction, SNP effect analyses, population genetic analyses etc."

Grammer tweaking